### PR TITLE
Adding a dev account in k3s

### PR DESCRIPTION
**Pull request by gemblerz**
The dev users who have access to the dev nodes need to launch their app using kubectl for test. This service account grants them to do so with the limitation that they only spin up pods within the namespace.
_WARNING: There is no resource constraints on this account_

This creates,
- `dev` namespace
- `node-dev-svc-account` user having all permissions on Pods within the namespace
- 2 services forwarding `wes-rabbitmq` and `wes-audio-server` to the namespace
- a kubeconfig at `/home/waggle/.kube/config` to use kubectl command for the user

This creation process is done manually (Please see kubernetes/tools/devaccount/deploy-dev-account.sh)

### CODE REVIEW

File: kubernetes/tools/devaccount/deploy-dev-account-payload.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+
+kubectl apply -k .
+
+mkdir -p /home/waggle/.kube
```
Comment (jswantek) [inline kubernetes/tools/devaccount/deploy-dev-account-payload.sh:5]: This implies that the `waggle` user already exists, and that basically the node is running 1.8.2 OS.

How do you want to handle this (if at all) on nodes that don't have this update yet?

File: kubernetes/tools/devaccount/deploy-dev-account-payload.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+cat <<EOF > /home/waggle/.kube/config
+apiVersion: v1
+clusters:
+contexts:
+current-context:
+kind: Config
+preferences: {}
+EOF
+
+chown waggle:waggle /home/waggle/.kube/config
```
Comment (jswantek) [inline kubernetes/tools/devaccount/deploy-dev-account-payload.sh:17]: it is probably better off to just do a `chown -r` at the end of `/home/waggle/.kube`.

Now if I remember right... when the user runs any commands on the nodes now with `kubectl` they have to do so as `sudo`.  and therefore the `.kube` folder is owned by `root`

see my `waggle` user on W030

```
root@ws-nxcore-000048B02D15BC68:/home/waggle# ls -la
total 48
drwxr-xr-x 7 waggle waggle 4096 Dec  5 02:49 .
drwxr-xr-x 1 root   root   4096 Dec  2 21:09 ..
-rw------- 1 waggle waggle  247 Dec  5 02:49 .bash_history
-rw-r--r-- 1 waggle waggle  220 Apr  4  2018 .bash_logout
-rw-r--r-- 1 waggle waggle 3771 Apr  4  2018 .bashrc
drwx------ 2 waggle waggle 4096 Dec  2 21:16 .cache
drwxr-xr-x 2 waggle waggle 4096 Apr 23  2021 Desktop
drwx------ 3 waggle waggle 4096 Dec  2 21:16 .gnupg
drwxr-x--- 3 root   root   4096 Dec  2 21:17 .kube
-rw-r--r-- 1 waggle waggle  807 Apr  4  2018 .profile
drwxr-xr-x 2 waggle waggle 4096 Dec  2 21:09 .ssh
-rw-r--r-- 1 waggle waggle 2090 Oct 16  2020 .xsessionrc
```

i therefore dont think the files need to be owned by `waggle`

File: kubernetes/tools/devaccount/deploy-dev-account-payload.sh (hunk 1 above)
Comment (jswantek) [reply kubernetes/tools/devaccount/deploy-dev-account-payload.sh:5]: I think you should only proceed on nodes that already have the `waggle` user.

**Review by jswantek — CHANGES_REQUESTED** [review]

File: kubernetes/tools/devaccount/deploy-dev-account-payload.sh (hunk 1 above)
Comment (gemblerz) [reply kubernetes/tools/devaccount/deploy-dev-account-payload.sh:9]: Yes, I added a user check. Thanks!

File: kubernetes/tools/devaccount/deploy-dev-account-payload.sh (hunk 2 above)
Comment (gemblerz) [reply kubernetes/tools/devaccount/deploy-dev-account-payload.sh]: The `/home/waggle/.kube` is owned by the Waggle user now.

**Comment by gemblerz** [comment]
@jswantek I made changes to reflect your comments. The only decision left is whether we do `sudo` to execute `kubectl` for waggle or not.

**Comment by gemblerz** [comment]
`wes-camera-provisioner` is updated and included in this PR, which could be a separated PR, but just added to make both simple.

**Pull request merged**